.github/workflows/build-test-deploy.yml

```diff
@@ -519,9 +519,8 @@ jobs:
     if: github.ref == 'refs/heads/main' || github.ref == 'refs/heads/staging' || github.ref == 'refs/heads/testnet-prod' || github.ref == 'refs/heads/prod'
     runs-on: ubuntu-latest
     env:
-      REGISTRY: 679752396206.dkr.ecr.us-east-1.amazonaws.com
-      IMAGE_TAG: ${{ github.sha }}
-      REPOSITORY: nxtp-cartographer
+      REGISTRY: ghcr.io
+      IMAGE_NAME: connext/sequencer-subscriber
```

Comment:
I fail to understand these changes here

Reply:
@rhlsthrm Was this a typo? What is sequencer image doing under carto?